bin/syclcc-clang

```diff
@@ -1665,7 +1665,7 @@ class compiler:
   @property
   def common_cxx_flags(self):
     args = [
-      "-I" + self._hipsycl_include_path,
+      "-isystem" + self._hipsycl_include_path,
```

Comment:
I thought that one would have to leave a space between `-isystem` and the path. :confused: But this does seem to work as-is. :+1: At least in my own test the warnings are gone with this included.

Reply:
hm.. good catch, I thought so too :P

Reply:
Just to confirm that the updated/fixed formalism also works as desired in my test.

Reply:
Thanks! :)